src/Lookup/TermLookupException.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+namespace Wikibase\DataModel\Services\Lookup;
+
+use Wikibase\DataModel\Entity\EntityId;
+
+/**
+ * @since 1.2
+ *
+ * @licence GNU GPL v2+
+ * @author Adam Shorland
+ */
+class TermLookupException extends \RuntimeException {
+
+	private $entityId;
+
+	public function __construct(
+		EntityId $entityId,
+		array $languageCodes,
```

Comment:
The array bit is a bit odd, since some of the methods just take one. I don't think this is a real issue, just saying it's a bit odd.